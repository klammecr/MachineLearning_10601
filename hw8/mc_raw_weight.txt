-2.625947261047713699e+00 1.376883210703988780e+00 -1.347997149996411209e-03
-2.616153094902327503e+00 1.373128451161174324e+00 1.073523810835094435e-03
-2.623065876639106619e+00 1.377416297855733829e+00 1.286453073423584172e-03
