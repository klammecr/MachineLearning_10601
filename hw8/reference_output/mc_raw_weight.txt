-2.616201458164875593e+00 1.372446352613752252e+00 -8.118122206508656064e-04
-2.624314255042218047e+00 1.377997342131049630e+00 -5.094025945611515866e-05
-2.624000732921778489e+00 1.378888252513226664e+00 1.757295801944946728e-03
